### DISCUSSION TITLE
0.9.0

### DISCUSSION THREAD

**Discussion by amusleh-spotware-com** [spotware/OpenApiPy#9] [Announcements]
**Full Changelog**: https://github.com/spotware/OpenApiPy/compare/0.8.0...0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/spotware/OpenApiPy/releases/tag/0.9.0'>0.9.0</a>.</em>

**Comment by Matx00** [comment]
Hi, 
are you planning to develop this package further to make it more as a "high-level" wrapper, i.e. with functions like send_order(accountId, API_key,...., tradeSide, orderType, lots, stopLoss, .....) or get_tick_price(accountId, symbolId)?
Thank you very much, very appreciated package!

**Reply by amusleh-spotware-com** [reply]
Hi,

No, we don't have such a plan, but the in it's current state it's already a high level wrapper.

You don't provide the data in method parameters but inside Protobuf objects.

This package allows you to easily create an API proto message and then send it.

**Reply by Matx00** [reply]
Hi,
Thank you for your answer!
I'm sorry, I admit I'm quite inexperienced. Could you then suggest to me a way to implement something that works synchronously, like "response = await send(message)"?

thank you again

**Reply by amusleh-spotware-com** [reply]
Hi,

The package uses Twisted which is asynchronous, it's not possible to use it synchronously, you just have to use Twisted deferreds and it works like await syntaxt.

Please check the samples, specifically the Jupyter or console sample, and you will understand how to use it.
